ims/common/bmi_config.py

```diff
@@ -0,0 +1,43 @@
+import ims.common.constants as constants
+
+
+def parse_config(cfg):
+    # Mandatory Options
+    # BMI Section
+    cfg.option(constants.BMI_SECTION, constants.UID_OPT)
+    cfg.option(constants.BMI_SECTION, constants.SERVICE_OPT)
+
+    # DB Section
+    cfg.option(constants.DB_SECTION, constants.DB_PATH_OPT)
+
+    # RPC Section
+    cfg.option(constants.RPC_SECTION, constants.RPC_SERVER_IP_OPT)
+    cfg.option(constants.RPC_SECTION, constants.RPC_SERVER_PORT_OPT, int)
+    cfg.option(constants.RPC_SECTION, constants.NAME_SERVER_IP_OPT)
+    cfg.option(constants.RPC_SECTION, constants.NAME_SERVER_PORT_OPT, int)
+
+    # TFTP Section
+    cfg.option(constants.TFTP_SECTION, constants.PXELINUX_PATH_OPT)
+    cfg.option(constants.TFTP_SECTION, constants.IPXE_PATH_OPT)
+
+    # REST API Section
+    cfg.option(constants.REST_API_SECTION, constants.REST_API_IP_OPT)
+    cfg.option(constants.REST_API_SECTION, constants.REST_API_PORT_OPT, int)
+
+    # Logs Section
+    cfg.option(constants.LOGS_SECTION, constants.LOGS_PATH_OPT)
+    cfg.option(constants.LOGS_SECTION, constants.LOGS_DEBUG_OPT, bool)
+    cfg.option(constants.LOGS_SECTION, constants.LOGS_VERBOSE_OPT, bool)
```

Comment:
We should make the data type a keyword arg or otherwise it'll be hard to change the API of the config parser in the future.

So this will be, 
cfg.option(constants.LOGS_SECTION, constants.LOGS_VERBOSE_OPT, type=bool)